tests/unit/maze_transformer/generation/test_generators.py

```diff
@@ -16,3 +18,35 @@ def test_gen_dfs_oblong():
     maze = LatticeMazeGenerators.gen_dfs(three_by_four)
 
     assert maze.connection_list.shape == (2, 3, 4)
+
+
+def test_gen_dfs_deterministic_with_seed():
```

Comment:
I might delete these, it just took me a minute to remember how seeds work and I wrote this in the meantim